apps/condo/domains/meter/schema/MeterReadingFilterTemplate.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema meter.MeterReadingFilterTemplate 'name:Text; employee:Relationship:OrganizationEmployee:CASCADE; filters:Json'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/meter/access/MeterReadingFilterTemplate')
+const { METER_READING_FILTERS_FIELD } = require('./fields/MeterReadingFilters')
+
+
+const MeterReadingFilterTemplate = new GQLListSchema('MeterReadingFilterTemplate', {
+    schemaDoc: 'Employee specific meter reading filter template',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        name: {
+            schemaDoc: 'Meter reading filter template name',
+            type: Text,
+            isRequired: true,
+        },
+
+        employee: {
+            schemaDoc: 'Link to employee, who created this template',
+            type: Relationship,
+            ref: 'OrganizationEmployee',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        filters: METER_READING_FILTERS_FIELD,
```

Comment:
If this is a set of fields for a declared template of a filter, therefore it's strange to have a filters for filter.
Maybe to name it `fields`?
```suggestion
        fields: METER_READING_FILTERS_FIELD,
```